src/SprykerEco/Zed/Computop/Persistence/Propel/Schema/spy_payment_computop.schema.xml

```diff
@@ -185,7 +185,7 @@
         <column name="x_id" required="false" type="VARCHAR" size="100"/>
         <column name="code" required="false" type="VARCHAR" size="20"/>
         <column name="status" required="false" type="VARCHAR" size="20"/>
-        <column name="description" required="false" type="VARCHAR" size="256"/>
+        <column name="description" required="false" type="VARCHAR" size="255"/>
```

Comment:
It looks like a major, was approved by a tech lead?

Reply:
Fixed

@sadkivskiy stated that this change could be reverted.